src/controllers/FulfillmentController.js

```diff
@@ -46,6 +46,16 @@ class FulfillmentController {
       callback(err);
     }
   };
+
+  saveMessage = async (req, res) => {
```

Comment:
This method and the one in the FulfillmentService need to be renamed to something else that isn't as generic as "saveMessage". Maybe something like processDialogFlowWebhook. All it's doing now is saving the message but it will soon be doing a lot more than just that.

Reply:
done